src/linagora.esn.unifiedinbox/app/controllers.js

```diff
@@ -481,6 +484,14 @@ angular.module('linagora.esn.unifiedinbox')
                 self.fixTime('toDate');
                 self.updateDateAndTime('toDate');
               }
+
+              Object.defineProperty(self.vacationDate, 'start', {
+                get() { return esnDatetimeService.format($scope.vacation.fromDate.toDate(), 'LLL') },
+              })
+
+              Object.defineProperty(self.vacationDate, 'end', {
```

Comment:
IMO, better if you have a util method to 2 `defineProperty` actions for `vacationDate`. And explain what are they for in the method doc.

Reply:
Done I use a factory for this. It's better